src/Controller/NewsController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace App\Controller;
+use App\Entity\News;
+use Symfony\Component\Finder\Finder;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Serializer\Encoder\XmlEncoder;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\HttpKernel\Kernel;
+// use SplFileInfo;
+
+// This controller renders the files in 'public/news'
+
+class NewsController extends AbstractController
+{
+    public function news()
+    {
+        // TODO: use CacheableSupportsMethodInterface
+        // TODO: seperate page for every news
+
+        return $this->render('news.html.twig', [
+            'newsList' => $this->getNewsList($this)
+        ]);
+    }
+
+    public function getNewsList($that)
```

Comment:
Nur ein Hinweis, aber nichts was wir hier ändern müssen. Aber man würde in echten Symfony-Projekten das hier nicht so implementieren (vllt. ist dir das teils bewusst, aber dennoch vielleicht etwas für dich zum Mitnehmen :D ):

- Er ist nicht klar definiert, muss aber eine AbstractController Instanz sein
- dadurch dass es eine AbstractController Instanz sein muss ist die Methode nicht sehr vielseitig einsetzbar (lieber Parameter explizit machen)
- Generell macht es keinen Sinn das dies in einem Controller ist, denn diese Methode managed keine View, es sollte ein Service sein und sollte ausschließlich die Information zurück liefern und nichts mit Twig machen, so dass wer auch immer den Service benutzt am Ende frei entscheiden kann was er mit den News-Items anstellt!